_assets/scss/elements/_typography.scss

```diff
@@ -78,6 +78,17 @@
   @include breakpoint($large) {}
 }
 
+@mixin paragraph {
```

Comment:
If it changes then the design team should call out that change in their type ramp. Type ramp is the source of truth here.